go/flags/endtoend/vtorc.txt

```diff
@@ -23,14 +23,14 @@ Usage of vtorc:
       --keep_logs duration                           keep logs for this long (using ctime) (zero to keep forever)
       --keep_logs_by_mtime duration                  keep logs for this long (using mtime) (zero to keep forever)
       --lameduck-period duration                     keep running at least this long after SIGTERM before stopping (default 50ms)
-      --lock-shard-timeout duration                  Duration for which a shard lock is held when running a recovery (default 30s)
```

Comment:
do we need to deprecate this flag?

Reply:
I have deprecated it - 
```
fs.Duration("lock-shard-timeout", 30*time.Second, "Duration for which a shard lock is held when running a recovery")
_ = fs.MarkDeprecated("lock-shard-timeout", "Please use lock-timeout instead.")
```

Deprecated flags don't show up in the flags output either.

Reply:
Added this to the summary too.